src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs

```diff
@@ -65,15 +63,34 @@ public async Task StartAsync(CancellationToken cancelToken = default)
                     new ChannelOption(ChannelOptions.MaxSendMessageLength, int.MaxValue),
                 };
 
-                if (this.grpcServer != null)
+                if (this.grpcServer is not null)
```

Comment:
Not sure this whole ShutdownAsync section is still necessary - given we know that it throws an IOException when 4001 is conflicted anyway, I doubt it is having any effect on global resources as originally theorized. Let me know if I should remove it alltogether

Reply:
I think you're probably right, TBH. However, it definitely looks suspicious that we don't do anything to dispose of resources. I suggest we keep the shutdown call but with the added exception handling, as you have in the current commit. That will at least remove all doubts.